src/tv2_offtube_showstyle/config-manifests.ts

```diff
@@ -182,6 +185,49 @@ export const gfxDesignTemplates: ConfigManifestEntry[] = [
 	}
 ]
 
+const GFX_SCHEMA_TABLE_ID = 'GfxSchemaTemplates'
+const GFX_SCHEMA_NAME_COLUMN_ID = 'GfxSchemaTemplatesName'
+
+export const gfxSchemaTemplates: ConfigManifestEntry[] = [
+	{
+		id: GFX_SCHEMA_TABLE_ID,
+		name: 'GFX Skema Templates',
+		description: 'The values for the Skema and Design combinations',
+		type: ConfigManifestEntryType.TABLE,
+		required: false,
+		defaultVal: [],
+		columns: [
+			{
+				id: GFX_SCHEMA_NAME_COLUMN_ID,
+				name: 'iNews Name',
+				description: 'The name of the design',
+				type: ConfigManifestEntryType.STRING,
+				required: false,
+				defaultVal: '',
+				rank: 0
+			},
+			{
+				id: 'INewsSkemaColumn',
+				name: 'iNews Skema Column',
+				description: 'The selected skema',
+				type: ConfigManifestEntryType.STRING,
+				required: false,
+				defaultVal: '',
+				rank: 1
+			},
+			{
+				id: 'VizTemplate',
+				name: 'Viz Template Name',
```

Comment:
There's no Viz in Qboxes, and I'm not sure what skema corresponds to in the HTML package, but if it is or will be used, the description could at least say something like `The name of the skema in the HTML package`

Reply:
Updated! The text, however, uses Jacobs suggestion from Slack.